src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/helper/builders/CourseAllocation.kt

```diff
@@ -15,15 +17,15 @@ import java.time.LocalDate
 @DslMarker
 annotation class CourseAllocationDslMarker
 
-@TestDataDslMarker
+@NomisDataDslMarker
 interface CourseAllocationDsl {
 
   @CourseAllocationPayBandDslMarker
   fun payBand(
     startDate: String = "2022-10-31",
     endDate: String? = null,
     payBandCode: String = "5",
-  )
+  ): OffenderProgramProfilePayBand
```

Comment:
I was able to convert courseAllocation, courseAllocationPayBand and courseAttendance to the new DSL style as they now live underneath the `newOffender` DSL.